cmd/plugin.go

```diff
@@ -864,3 +869,28 @@ func getConnectionState(ctx context.Context) (steampipeconfig.ConnectionStateMap
 
 	return connectionStateMap, res
 }
+
+func cleanupOldTmpDirs() {
```

Comment:
Move this function into [pkg/plugin](https://github.com/turbot/steampipe/tree/9754ed0c1a84e996d820dfe75199131046d83727/pkg/plugin). Create a new file `cleanup.go` and move the function there.